modules/network-observability-operator-install.adoc

```diff
@@ -0,0 +1,28 @@
+// Module included in the following assemblies:
+
+// * installing-operators.adoc
+
+:_content-type: PROCEDURE
+[id="network-observability-operator-installation_{context}"]
+
+= Installing the Network Observability operator
+You can install the Network Observability operator using the {product-title} web console Operator Hub.
+
+.Procedure
+ 
+.. In the {product-title} web console, click *Operators* -> *OperatorHub*.
+//In the Operator Hub on the test console Julian provided perms for, its NetObserv Operator, with a capital O. 
+.. Choose  *Network Observability operator* from the list of available Operators in the *OperatorHub*, and click *Install*.
+.. Navigate to *Operators* -> *Installed Operators*. Under Provided APIs for Network Observability, select the *Flow Collector* resource.
+. Navigate to the *Flow Collector* tab, and click *Create FlowCollector*. Make the following selections in the form view:
++
+- *Sampling* : Specify a sampling size that will not consume too many resources. For more information, see the FlowCollector custom resource configuration parameters reference table. 
+- *deploymentModel*: If you are using Kafka, verify Kafka is selected.
+- *Loki.url*: Since authentication is specified separately, this URL needs to be updated to `https://loki-gateway.gateway-http.netobserv.svc:8080/api/logs/v1/network`
+-*authToken*: Select the `HOST` value.
+-*tls.caCert*: Remove the existing values in the `certFile` and `name` fields.
+-*tls.enable* and *tls.insecureSkipVerify*: Check the boxes for both of these specifications.
```

Comment:
It doesn't look right, we should document using the certificates, and not in "insecure" mode.
So:
- `tls.enable` must be checked

And that's all, `caCert` and `insecureSkipVerify` can be left with their defaults.

Reply:
@memodi / @stleerh / @jpinsonneau can you confirm the above is correct with 5.6? (I tested with 5.5; except the tenant stuff, the URLs haven't changed between 5.5 and 5.6?)

Reply:
@skrthomas in fact, I just realized that the information here is actually correctly described in the "Configuring [...]" section, so wouldn't it be simpler to remove these explanations on configuring FlowCollector here, and just mention and/or link to the next "Configuring [...]" page?

Reply:
@jotak confirming URLs are correct, however I run into `x509: certificate signed by unknown authority fields.level` when using default CA bundle `loki-ca-bundle` on FLP side, I wonder whether we should document this error in troubleshooting doc and suggest to set `insecureSkipVerify` to true ?

Reply:
@memodi sure we can document this issue in troubleshooting and/or known issues. But I wouldn't like to document here `insecureSkipVerify=true` as a default or normal thing - we must promote security by default.

I wonder if you might have an issue about certificate issuer in the specific ocp build you're using? Did you try something different, e.g. with cluster-bot + a release candidate ?

Reply:
@jotak I haven't different launching cluster in a different way, but can try that later. I am also hitting different issue even when I have `insecureSkipVerify=true` https://coreos.slack.com/archives/C02939DP5L5/p1671623085801599

Reply:
@xenolinux FYI potential Troubleshooting addition.

Reply:
@jotak I think I still want to leave these here because it is the same, but different in that here I was describing what to do when you're creating the FlowCollector and in the Configuring... describes how to view and edit an existing FlowCollector.

Reply:
@jotak As far as advising against using `querierUrl`..I'm deliberating the best place to do that. You say not set the `querierUrl` when using the Loki Operator, so is this only a limitation if you install Loki using the Loki Operator? But not if you are using Loki via another type of installation?

Reply:
@jotak as far as noting about `querierUrl` I'm wondering if I should duplicate this note also in the API reference and here just in case. Seems like we don't want users doing this for sure so better to have it multiple places, even if they are more likely to see it in the Install than the reference table.

Reply:
I added a link to the Resource and the API Reference, but I have to do it in a weird way due to documentation linking limitations, so its now in Additional Resources at the bottom of the Installing Network Observability operator topic: https://53263--docspreview.netlify.app/openshift-enterprise/latest/networking/network_observability/installing-operators.html#network-observability-operator-installation_network_observability